### ISSUE TITLE
Adding overlays with pixel coordinates (px, py) while adding image not working

### ISSUE THREAD

**Issue by sepastian** [openseadragon/openseadragon#742]
I am opening a new issue for this, @iangilman, as it has nothing to do with #507.

Started looking into this and I'm stuck with the `addOverlay` function in `viewer.js`. When procesing as single tile source, `addOverlay` gets invoked for each overlay.

```
addOverlay: function( element, location, placement, onDraw ) { ... }
```

What do I need to specify for `element`, `location`, `placement`, `onDraw`? Could you point me in the right direction?

Thanks!

**Comment by avandecreme** [comment]
Using this 4 parameters is legacy. This is handled here: https://github.com/openseadragon/openseadragon/blob/master/src/viewer.js#L1761-L1770
So you can just pass the options object directly as it is currently done.

**Comment by iangilman** [comment]
@sepastian Thanks for starting a new issue. And yes, what @avandecreme said. This is how we do it here: 

https://github.com/openseadragon/openseadragon/blob/master/src/viewer.js#L632-L638

**Comment by sepastian** [comment]
Thank for your comments, I have created PR #745. The only change required was what @iangilman suggested initially.

The issue I reported is gone now, all tests are passing. However, since I don't know much about the internals of openseadragon, someone else may need to make sure everything is behaving as expected.

**Comment by iangilman** [comment]
Awesome; I guess this answers my question from #745. I'll kick the tires and merge. Thanks again!

**Comment by iangilman** [comment]
Fixed by #745
